### DISCUSSION TITLE
feature request: Option to set a server as public, with member-only channels

### DISCUSSION THREAD

**Discussion by Freeplayg** [stoatchat/discussions#477] [Feature Suggestions]
### What do you want to see?

Have the option for setting a server as public, meaning that anyone can view the server without an account

The servers list could also possibly have log in / create an account icon buttons when logged out

### PWA

- [ ] Yes, this feature request is specific to the PWA.

**Comment by ryo33** [comment]
This feature makes an organization that has motivations for openness and transparency such as open-source community can use this impressive software for their internal or external communications. Public searchable could be a critical option for them (and mine's). I know several open-software that support it (I can list them here), but I personally prefer this software for my own reasons.
